.cirrus.yml

```diff
@@ -105,7 +105,10 @@ sonar_scan_task:
   depends_on:
     - validation
     - unit_test
+  env:
+    SONAR_SCANNER_NPM_VERSION: 3.6.0-2097
   script:
+    - npm install --no-save "https://$ARTIFACTORY_PRIVATE_USERNAME:$ARTIFACTORY_PRIVATE_PASSWORD@repox.jfrog.io/artifactory/sonarsource-npm-public-dev/sonarqube-scanner/-/sonarqube-scanner-$SONAR_SCANNER_NPM_VERSION.tgz"
```

Comment:
Let's do that in the docker file instead? And install it as global maybe?

Reply:
Yeah I can do that, I think it aligns more with how our users install it 👍🏻

Reply:
docker build task doesn't have access to the artifactory credentials via Vault to have this work. After dogfooding this will work because we will install from npm (not artifactory)